plugin/pkg/scheduler/scheduler.go

```diff
@@ -89,6 +89,12 @@ func (s *Scheduler) Run() {
 
 func (s *Scheduler) scheduleOne() {
 	pod := s.config.NextPod()
+	if pod.DeletionTimestamp != nil {
+		// TODO: Consider delete this kind of pods
```

Comment:
Delete this TODO since it maybe dangerous and not the responsibility of scheduler.